packages/dnb-eufemia/src/elements/img/Img.tsx

```diff
@@ -47,7 +47,7 @@ const Img = ({
         as="img"
         alt={alt}
         internalClass={classnames(
-          'dnb-img__img',
+          'dnb-img',
           hasError && 'dnb-img__img--error'
```

Comment:
But maybe, we can remove it, and also change the modifier to dnb-img--error' instead?

```suggestion
          hasError && 'dnb-img--error'
```

Reply:
Hmm, I really have no Idea of what is best to do here 😄

Reply:
I would suggest doing the latter. I can't find any good reasons to have a extra identifier. Its an internal style also.